lib/opentracing/span.rb

```diff
@@ -0,0 +1,60 @@
+module OpenTracing
+  # Span represents an OpenTracer Span
+  #
+  # See http://www.opentracing.io for more information.
+  class Span
+    NOOP_INSTANCE = Span.new.freeze
+    # Name of the operation
+    attr_accessor :operation_name
+
+    # Span Context
+    attr_reader :span_context
+
+    # Tags on the span
+    attr_reader :tags
```

Comment:
read access to tags is not part of the API (makes noop impls cheaper to implement)